components/device-mgt/org.wso2.carbon.device.mgt.core/pom.xml

```diff
@@ -359,6 +359,12 @@
 			<groupId>javassist</groupId>
 			<artifactId>javassist</artifactId>
 		</dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
+            <version>${power.mock.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Shall we move this dependency to parent pom, if it is not already there and remove the version from this pom